dfsclient/src/main/java/org/hpccsystems/dfs/client/PlainConnection.java

```diff
@@ -0,0 +1,616 @@
+                                + sock.getPort());
+                    }
+                    else
+                    {
+                        SocketFactory sf = SocketFactory.getDefault();
+                        sock = sf.createSocket();
+
+                        // Optimize for bandwidth over latency and connection time.
+                        // We are opening up a long standing connection and potentially reading a significant amount of
+                        // data
+                        // So we don't care as much about individual packet latency or connection time overhead
+                        sock.setPerformancePreferences(0, 1, 2);
+                        sock.connect(new InetSocketAddress(this.getIP(), this.dataPart.getPort()),
+                                DEFAULT_CONNECT_TIMEOUT_MILIS);
+                    }
+                    log.debug("Connected: Remote address = " + sock.getInetAddress().toString() + " Remote port = "
+                            + sock.getPort());
+                }
+                catch (java.net.UnknownHostException e)
+                {
+                    throw new HpccFileException("Bad file part addr " + this.getIP(), e);
+                }
+                catch (java.io.IOException e)
+                {
+                    throw new HpccFileException(e);
+                }
+
+                try
+                {
+                    this.dos = new java.io.DataOutputStream(sock.getOutputStream());
+                    this.dis = new java.io.DataInputStream(sock.getInputStream());
+                }
+                catch (java.io.IOException e)
+                {
+                    throw new HpccFileException("Failed to create streams", e);
+                }
+                this.active = true;
+                try
+                {
+                    String readTrans = makeInitialRequest();
+                    int transLen = readTrans.length();
+                    this.dos.writeInt(transLen);
+                    this.dos.write(readTrans.getBytes(HPCCCharSet), 0, transLen);
+                    this.dos.flush();
+                }
+                catch (IOException e)
+                {
+                    throw new HpccFileException("Failed on initial remote read read trans", e);
+                }
+                return;
+            }
+            catch (Exception e)
+            {
+                log.error("Could not reach file part: '" + dataPart.getThisPart() + "' copy: '"
+                        + (currentFilePartCopyIndex + 1) + "' on IP: '" + getIP());
+                log.error(e.getMessage());
+
+                if (!setNextFilePartCopy())
+                    throw new HpccFileException("Unsuccessfuly attempted to connect to all file part copies", e); // this
+                                                                                                                  // should
```

Comment:
this is prob a side effect of the code format template that needs to be cleaned up